Configuration/TCA/Overrides/sys_template.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-defined('TYPO3_MODE') || die();
+defined('TYPO3') || die();
```

Comment:
I doubt this will work in older versions of TYPO3.

Reply:
damn, you are right...